satpy/etc/readers/eps_sterna_mwr_l1b_nc.yaml

```
@@ -0,0 +1,532 @@
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_zenith_horn1:
+        name: satellite_zenith_horn1
+        file_type: eps_sterna_l1b_nc
+        file_key: data/navigation/satellite_zenith_angle
+        standard_name: satellite_zenith_angle
+        horn: "1"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_zenith_horn2:
+        name: satellite_zenith_horn2
+        file_type: eps_sterna_l1b_nc
+        file_key: data/navigation/satellite_zenith_angle
+        standard_name: satellite_zenith_angle
+        horn: "2"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_zenith_horn3:
+        name: satellite_zenith_horn3
+        file_type: eps_sterna_l1b_nc
+        file_key: data/navigation/satellite_zenith_angle
+        standard_name: satellite_zenith_angle
+        horn: "3"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_zenith_horn4:
+        name: satellite_zenith_horn4
+        file_type: eps_sterna_l1b_nc
+        file_key: data/navigation/satellite_zenith_angle
+        standard_name: satellite_zenith_angle
+        horn: "4"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_azimuth_horn1:
+        name: satellite_azimuth_horn1
+        file_type: eps_sterna_l1b_nc
+        file_key: data/navigation/satellite_azimuth_angle
+        standard_name: satellite_azimuth_angle
+        horn: "1"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_azimuth_horn2:
+        name: satellite_azimuth_horn2
+        file_type: eps_sterna_l1b_nc
+        file_key: data/navigation/satellite_azimuth_angle
+        standard_name: satellite_azimuth_angle
+        horn: "2"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_azimuth_horn3:
+        name: satellite_azimuth_horn3
+        file_type: eps_sterna_l1b_nc
+        file_key: data/navigation/satellite_azimuth_angle
+        standard_name: satellite_azimuth_angle
+        horn: "3"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_azimuth_horn4:
+        name: satellite_azimuth_horn4
+        file_type: eps_sterna_l1b_nc
+        file_key: data/navigation/satellite_azimuth_angle
+        standard_name: satellite_azimuth_angle
+        horn: "4"
+        coordinates:
+            - longitude
+            - latitude
+
+file_types:
+    eps_sterna_l1b_nc:
+        # W_XX-EUMETSAT-Darmstadt,SAT,AWS1-MWR-1B-RAD_C_EUMT_20241121085911_G_D_20241109234502_20241110004559_T_N____.nc
+        file_reader: !!python/name:satpy.readers.mwr_l1b.AWS_EPS_Sterna_MWR_L1BFile
+        file_patterns: [
+        'W_XX-EUMETSAT-Darmstadt,SAT,{platform_name}-MWR-1B-RAD_C_EUMT_{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_{disposition_mode:1s}_{processing_mode:1s}____.nc',
```

Hard-coded processing center etc.
```suggestion
        'W_XX-{organization:s}-{location:s},SAT,{platform_name}-MWR-1B-RAD_C_{organization2:s}_{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_{disposition_mode:1s}_{processing_mode:1s}____.nc',
```

---

Fixed